ios/CovidShield/ExposureNotification.m

```diff
@@ -179,6 +179,7 @@ - (void)invalidate
     NSNumber *idx = @(self.reportedSummaries.count);
     [self.reportedSummaries addObject:summary];
     resolve(@{
+      @"attenuationDurations": summary.attenuationDurations,
```

Comment:
should this be: `@(summary. attenuationDurations),`?

Reply:
No, because `summary.attenuationDurations` is an array, so it doesn't need `@()`.